PrebidMobile/API1.0/src/test/java/org/prebid/mobile/PrebidServerAdapterTest.java

```diff
@@ -148,7 +157,13 @@ public void testInvalidPrebidServerConfigIdForRubiconHostedPrebidServer() {
         adapter.requestDemand(requestParams, mockListener, uuid);
         Robolectric.flushBackgroundThreadScheduler();
         Robolectric.flushForegroundThreadScheduler();
-        verify(mockListener).onDemandFailed(ResultCode.INVALID_CONFIG_ID, uuid);
+
+        ArgumentCaptor<ResultCode> resultCodeAC = ArgumentCaptor.forClass(ResultCode.class);
+        ArgumentCaptor<String> uuidAC = ArgumentCaptor.forClass(String.class);
+
+        verify(mockListener).onDemandFailed(resultCodeAC.capture(), uuidAC.capture());
+        assertThat(resultCodeAC.getValue(), Matchers.either(Matchers.is(ResultCode.INVALID_CONFIG_ID)).or(Matchers.is(ResultCode.PREBID_SERVER_ERROR)));
```

Comment:
Can we make similar changes for INVALID_CONFIG_ID as well?

Reply:
@avohraa, I have updated the PR